src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -302,6 +303,23 @@ export default {
 			}
 		},
 	},
+
+	// TODO: move the implementation to @nextcloud-vue/NcListItem
```

Comment:
Waiting for discussion of nextcloud/nextcloud-vue#3687, if the same approach could be implemented there